### PR TITLE
Add the initial metrics section overview

### LINKED ISSUE #1568: content: Intro to metrics
**Author**: @philipwalton 
**Reviewer**: @kaycebasques 

**Target publish date:** 2019-11-06

- [ ] Check this box if this is a hard deadline.
- [ ] Check this box if you'd like your reviewer to fix grammar and punctuation issues for you.

**Process**

- [x] 1. I've done my proposal
- [x] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a [draft](https://docs.google.com/document/d/1mQzjSTNKs6_yoOzVfaPZma7Uh6KzuOnb-OmCKIj7R5w/edit) to my proposal doc
- [ ] 4. Draft approved — _reviewer checks this_
- [ ] 5. I've submitted a pull request #<add PR number>
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by philipwalton**
Fixes #1568, fixes: #1575

This PR adds the initial Metrics section overview page as well as the content for the "User-centric metrics" doc as well as the "Custom metrics" doc.

In a follow-up PR I'll add content for each of the individual metrics docs (which includes updating/renaming some of our existing metrics docs).

_**Note:** I'm currently using a Lighthouse illustration as a placeholder for the Metrics section illustration. Once the new illustration is added, I'll update, but DO NOT MERGE until that happens._

### CODE REVIEW

File: src/site/content/en/metrics/user-centric-metrics/index.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+- A site might _appear_ to load quickly but then respond slowly (or not at all)
+  to user interaction.
+
+So when talking about performance, it's important to be precise and to refer to
+performance in terms of objective criteria that can be quantitatively measured.
+These criteria are known as _metrics_.
+
+But just because a metric is based on objective criteria and can be
+quantitatively measured, it doesn't necessarily mean those measurements are
+useful.
```
Comment (kaycebasques) [inline src/site/content/en/metrics/user-centric-metrics/index.md]: Are you still happy with this intro, given what you said in our chat?

> BTW, I'm realizing that I too often describe metrics as a means to measure performance, so I'm trying to go through and reframe certain parts (specifically in the custom metrics doc) to frame them as a tool to measure the user experience.

This intro heavily frames metrics as something related to performance

File: src/site/content/en/metrics/user-centric-metrics/index.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+For example, a server could respond with a minimal page that "loads" immediately
+but then defers fetching content and displaying anything on the page until
+several seconds after the `load` event fires. While such a page might
+technically have a fast load time, that time would not correspond to how a user
+actually experiences the page loading.
+
+Over the past few years, members of the Chrome team—in collaboration with the
+[W3C Web Performance Working Group](https://www.w3.org/webperf/)—have been
+working to standardize a set of new APIs and metrics that more accurately
+measure how users experience the performance of a web page.
```
Comment (kaycebasques) [inline src/site/content/en/metrics/user-centric-metrics/index.md]: Another instance of defining metrics as measuring performance

File: src/site/content/en/metrics/user-centric-metrics/index.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: User-centric metrics
```
Comment (kaycebasques) [inline src/site/content/en/metrics/user-centric-metrics/index.md]: Another option might be to rename this `Performance metrics` (the directory name should change too if you rename the title)

File: src/site/content/en/metrics/custom-metrics/index.md [hunk 4]
```diff
@@ -0,0 +1,377 @@
...
+Historically web developers haven't had many low-level APIs to measure
+performance, and as a result they've had to resort to hacks in order to measure
+whether a site was performing well.
+
+For example, it's possible to determine whether the main thread is blocked due
+to long-running JavaScript tasks by running a `requestAnimationFrame` loop and
+calculating the delta between each frame. If the delta is significantly longer
+than the display's framerate, you can report that as a long task. Such hacks are
+not recommended, though, because they actually affect performance themselves
+(e.g. drain battery).
```
Comment (kaycebasques) [inline src/site/content/en/metrics/custom-metrics/index.md]: When we're happy with all the content I'll do a copyedit to remove this `e.g.` and the others I didn't catch. Leave this comment unresolved for me.

**Comment by philipwalton** [comment]
Taking another look, I agree that it's not realistic (given the current content) to remove the performance aspect and frame it entirely around user experience.

I've updated the title to add "Performance" back in, and I've updated the URLs as well.

I've also added the "metrics" illustration, so I'm removing the DO NOT MERGE label, as this can be merged as soon as the content is ready.

**Pull request merged**